porthtml.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+
+  <title>ibrahim Portfolio</title>
+ 
+ 
+</head>
+
+<body class="body1">
```

Comment:
- [ ] Please try to keep all the margins as in the given template. Here, your logo and the hamburger menu are not following the template, as in the given design both of them have some margin which makes them not to be right on the border of the page, so kindly fix this issue. The same thing happens with the section, so please make sure to fix it there too.